_docs/setup/kubernetes/sidecar-injection.md

```diff
@@ -132,7 +132,7 @@ See the Kubernetes [quick start]({{home}}/docs/setup/kubernetes/quick-start.html
 
 Note that unlike manual injection, automatic injection occurs at the pod-level. You won't see any change to the deployment itself. Instead you'll want to check individual pods (via `kubectl describe`) to see the injected proxy.
 
-#### Installing the webhook
+## Installing the webhook
```

Comment:
I'm pretty sure installing the webhook applies only to automatic injection and thus should be nested under automatic injection cc @ayj

Reply:
`Installing the webhook`, `Deploying an app`,  and ` Understanding what happened` should be nested under `Automatic sidecar injection`.

cc @yusuoh